docs/site-readme.md

```diff
@@ -151,4 +151,4 @@ Any questions, please ask one of the Clowder development team
 * [@psav](https://github.com/psav)
 * [@bsquizz](https://github.com/bsquizz)
 * [@bennyturns](https://github.com/bennyturns)
-* [@adamrdrew](https://github.com/adamrdrew)
+* [@adamrdrew](https://github.com/adamrdrew)
```

Comment:
What's this change?

Reply:
came with the make pre-push, doesn't do anything looks like